### ISSUE TITLE
Add CTA to 404 page: Donate.

### ISSUE THREAD

**Issue by stephaniehobson** [mozilla/bedrock#14209]
The 404 page includes three links that give users a opening to explore the site. MoFo would like us to add a 4th - Donate. A link to https://foundation.mozilla.org/en/donate/ can be used for now.

I think we need to be a little careful to use wording that makes it clear the donation is for MoFo.

**Comment by ehot** [comment]
@stephaniehobson could you please send me a screenshot of what the page currently looks like so that I can make sure I'm looking at the right page?

**Comment by ehot** [comment]
**Possible Icons:**
https://github.com/mozilla/protocol-assets/blob/main/icons/developer-innovations.svg
https://github.com/mozilla/protocol-assets/blob/main/icons/default-browser.svg

**Possible Wording:** 
Donate to Mozilla Foundation. We're proudly non-profit.

**Wording Based on:** 
https://foundation.mozilla.org/en/?form=donate

**Aside:** 
Is it just me or is this icon blue not black? https://www.mozilla.org/media/protocol/img/logos/firefox/logo-flat.0b1c84e06139.svg

<img width="795" alt="Screenshot 2024-02-15 at 3 30 51 PM" src="https://github.com/mozilla/bedrock/assets/2040610/85e22861-3c09-40fe-b874-8b162d0589d5">

**Comment by stephaniehobson** [comment]
@ehot it's not just you, that icon is some kind of deep purple/blue

I like the heart over the dino (and I think you do to because you used it in the mockup 😉 )

I think this is dev ready. Thanks @ehot!

**Comment by stephaniehobson** [comment]
@enavajas can you get legal to glance at this please?

**Comment by enavajas** [comment]
@stephaniehobson i have reached out to legal for guidance. will update once i hear back!

**Comment by stevejalim** [comment]
I just noticed that the careers section has its own 404 page, too https://www.mozilla.org/en-US/careers/position/gh/asdasdd/ - there's a `careers/404.html` template for that

**Comment by stephaniehobson** [comment]
We should update the careers/404 page too 💯

**Comment by michaelwhitney13** [comment]
👋 Hi all, Michael Whitney here from Digital Engagement at MoFo. So excited to see this moving forward. Thank you so much. I just wanted to add two things:

1. Can you please use this link for directing people to donate from the mozilla.org 404 pages? https://foundation.mozilla.org/?form=donate-404&utm_source=mozilla.org&utm_campaign=moco&utm_medium=referral&utm_content=404

2. What languages will this be translated in? Our fundraising platform supports localized content in English, Chinese, Danish, Dutch, Finnish, French, German, Hungarian, Italian, Japanese, Korean, Norwegian, Portuguese, Spanish, Swedish, and we will use our translation resources to get the copy updated.

And re: above, we have usually added utm_term=en, or whatever 2 character language code is present on the page.

**Comment by stephaniehobson** [comment]
@michaelwhitney13 👋 

Thank you for the link with utms! If it's easy we will add the language code (but I don't want to slow down adding it by making it too complicated).

[The 404 page is active in these locales](https://github.com/mozmeao/www-l10n/blob/master/metadata/404.json#L17) but it depends on the community for when/if it will be translated.

**Comment by michaelwhitney13** [comment]
Thanks @stephaniehobson! I just spoke with our fundraising operations person and he has an update on the link, I jumped ahead of him. He'll be back on Monday and I'll update then. Thank you.

**Comment by michaelwhitney13** [comment]
OK, this is the link we should use (no UTMs, just this): https://foundation.mozilla.org/?form=donate-404

And as we do more of this work, please reach out to me for any content, strategy, or link needs! Very excited to see this happening. Thank you.

**Comment by enavajas** [comment]
hi @stephaniehobson, i heard back from legal with the below information:

> To clarify: this would just be for the 404 pages on Mozilla.org? This doesn't affect Firefox, [foundation.mozilla.org](http://foundation.mozilla.org/), or other websites?

> If so, and we're just changing the 404 page for Mozilla.org itself, I think we can include a donation link. However, we'll need to adjust the language a little bit. Because Mozilla.org is primarily a MoCo channel ([foundation.mozilla.org](http://foundation.mozilla.org/) is MoFo's main channel), we'll need to be a bit careful about how we phrase who the donation is for. "We're proudly nonprofit" could look like a suggestion that MoCo, the company operating Mozilla.org and the services on it is claiming to be nonprofit. So we generally recommend something like "We're backed by a non-profit, the Mozilla Foundation. Donate today" or "Donate to the Mozilla Foundation,  the non-profit behind Firefox." We try to avoid simple "we" or "us" when specifically claiming non-profit status if it could be misconstrued, and especially when talking about donations.

**Comment by michaelwhitney13** [comment]
That's a good reminder, thank you. We need to save mentioning Firefox for surfaces where we can A/B test and where we'll be able to get significance. For this case then with that guidance from legal, we'd like to use:

> Donate to the Mozilla Foundation and reclaim the internet from big tech.

**Comment by janbrasna** [comment]
Reflecting the latest copy above, and the original request to leave out the "Explore" item (so with "Donate" we still have three), is this the intended outcome? In this order:

> ![404s](https://github.com/mozilla/bedrock/assets/1784648/1888d8f2-3294-4b3d-8403-0b1d18ae74c0)

@ehot Thanks for looking up the "heart" icon, I've digged through the assets twice and TBH would never find it under `"default-browser"` ;D

Normally there's a helper function for donate links:
https://github.com/mozilla/bedrock/blob/2c8907973d3016f637a9ee32de6771c6ddc00648/bedrock/mozorg/templatetags/misc.py#L379-L389
but since this one has a different form parameter and should not include tags the helper won't be used in this case then.

**Comment by alexgibson** [comment]
@michaelwhitney13 can you please confirm this copy is good to go? I think once approved we're ready to close this out as fixed. Thanks

![image](https://github.com/mozilla/bedrock/assets/400117/15366f81-f375-47b3-9adb-679243f5d270)

**Comment by michaelwhitney13** [comment]
Yes good on language, and confirming the link should be https://foundation.mozilla.org/?form=donate-404
